src/main/kotlin/app/batch/S3DirectoryWriter.kt

```diff
@@ -154,18 +156,18 @@ class S3DirectoryWriter(private val keyService: KeyService,
     private var maxBatchOutputSizeBytes: Int = 0
 
     @Value("\${aws.region}")
-    private var region: String = "eu-west-1"
+    private lateinit var region: String
 
     @Value("\${s3.bucket}")
     private lateinit var s3BucketName: String // i.e. "1234567890"
 
     @Value("\${s3.prefix.folder}")
     private lateinit var s3PrefixFolder: String //i.e. "mongo-export-2019-06-23"
 
-    @Value("\${source.table.name}")
-    private lateinit var tableName: String // i.e. "db.user.data"
+    @Value("\${topic.name}")
+    private lateinit var topicName: String // i.e. "db.user.data"
 
-    @Value("\${compress.output:false}")
+    @Value("\${compress.output:true}")
     private var compressOutput: Boolean = true
```

Comment:
You need to explain to me one day why you need a default both in the declaration AND in the annotation.

Reply:
You don't, but native types can't be latent-ed, so you have to put a value